demo-app/src/ClipPathPreviewForm.js

```diff
@@ -0,0 +1,67 @@
+import React from "react";
```

Comment:
When we come to the dialog box next time, it remembers the last input. It should start fresh everytime

Reply:
It should be able to start fresh now each time